sys-utils/libcmt/src/io-mock.c

```diff
@@ -219,6 +219,14 @@ static int mock_tx_exception(cmt_io_driver_mock_t *me, struct cmt_io_yield *rr)
     return store_next_output(me, "exception-", &me->exception_seq, rr);
 }
 
+static int mock_tx_gio(cmt_io_driver_mock_t *me, struct cmt_io_yield *rr) {
```

Comment:
Do you think this is sufficient?

I was under the impression that we should implement a way to feed the GIO response back in.
I tried doing this in this draft: https://github.com/cartesi/machine-emulator-tools/pull/35.

Reply:
I think the library still should have something regardless of the Rust http server.
The GIO response doesn't necessarily have to be related to the request.
But to be able to test the response path, even the basic stuff.
A gio call should change values in `cmt_io_yield` and put something on rx buffer.